### ISSUE TITLE
PowerShell Module Browser:https://docs.microsoft.com/en-us/powershell/module/

### ISSUE THREAD

**Issue by student-forever** [MicrosoftDocs/feedback#560]
Hello,

The PowerShell Module browser, doesn't include commands from the Exchange Online PowerShell module :

For example "Get-Mailbox" doesn't show any results.

Regards,

Ipsito Dutta.

**Comment by Powerhelmsman** [comment]
@EdisonDa Can you take a look at this issue?

**Comment by EdisonDa** [comment]
Thank you @student-forever for your reporting. The exchange online moniker has not been lived, we will let you know when we onboard this. Currently you can go to https://docs.microsoft.com/en-us/powershell/exchange/exchange-online/exchange-online-powershell?view=exchange-ps and filter "Get-Mailbox" as a work around. 

+@dstrome for awareness.

**Comment by student-forever** [comment]
Hey Edison,

Thank you for the post. However, from the dropdown to show the list of all
available PS modules, I do see that Exchange Powershell is available.
Please confirm what that is for?

**Comment by EdisonDa** [comment]
Yes. There are some moniker issues we are working on now. Currently, we can find the "Exchange" display in the module browser, but we are expect there is no modules/cmdlets displaying there. We will get you posted when it's ready to use.
